front/lib/api/assistant/configuration.ts

```diff
@@ -131,7 +131,13 @@ export async function searchAgentConfigurationsByName(
       },
     },
   });
-  return agentConfigurations || [];
+  const r = (
+    await Promise.all(
```

Comment:
IIRC, you can directly call the `getAgentConfigurations` with all the ids to avoid the need for a promise.all wrapper here.

Reply:
Don't seems like it accepts a list of ids.

Reply:
I meant for `getAgentConfigurations`  which is the method used under `getLightAgentConfiguration`:

https://github.com/dust-tt/dust/blob/fc26a61b098d0352a3b12222f945c87057b15c5c/front/lib/api/assistant/configuration.ts#L149-L159